etc/xdg/weston/weston.ini

```diff
@@ -2,6 +2,10 @@
 xwayland=true
 idle-time=0
 
+[output]
+name=eDP-1
```

Comment:
For our purposes short-term, this is fine.  I wonder if there's a way we can dynamically find the name of the built-in display. I have some devices where it may be eDP-2 or another device name entirely.

Reply:
Yeah, definitely a hack. I could not find a way to apply scaling globally.